src/api/rcos/users/mod.rs

```diff
@@ -79,6 +79,11 @@ impl UserRole {
         self == UserRole::Sysadmin || self == UserRole::FacultyAdvisor
     }
 
+    //TEMPORARY
```

Comment:
👀

Reply:
Not only was this temporary, it's already been superseded and is no longer used in any code I wrote. Can delete the following function. Nice catch